app/Http/Controllers/GenericResourceController.php

```diff
@@ -251,6 +251,12 @@ public function archive(Request $request)
      */
     public function unArchive(Request $request)
     {
+        $modelCollection = GenericModel::getCollection();
+
+        if (!strpos($modelCollection, '_archived')) {
```

Comment:
idea was to make routes `<collection>/<id>/archive` and `<collection>/<id>/unarchive`. I.e.
`/projects/aaaaaaaaaaaaaaaaaaaaaaaa/archive` and `/projects/aaaaaaaaaaaaaaaaaaaaaaaa/unarchive`. 

Current implementation requires that to be:
`/projects/aaaaaaaaaaaaaaaaaaaaaaaa/archive`
`/projects_archived/aaaaaaaaaaaaaaaaaaaaaaaa/archive`